scripts/std-lib-regression.sh

```diff
@@ -50,8 +50,7 @@ cd std_lib_test
 
 # Add some content to the rust file including an std function that is non-generic.
 echo '
-#[kani::proof]
```

Comment:
Why was this removed?

Reply:
So we don't have to manually inject Kani dependency here. Since our goal for this script is to make sure we can generate code for the standard library, this seems much easier to maintain than duplicating a bunch of the compiler flags here.